plugin/src/main/java/org/opensearch/ml/task/MLPredictTaskRunner.java

```diff
@@ -143,7 +143,13 @@ public void dispatchTask(
                 if (clusterService.localNode().getId().equals(node.getId())) {
                     log.debug("Execute ML predict request {} locally on node {}", request.getRequestID(), node.getId());
                     request.setDispatchTask(false);
-                    executeTask(request, listener);
+                    run(
```

Comment:
How about just add `checkOpenCircuitBreaker(mlCircuitBreakerService, mlStats);` in `executeTask` method ?

Reply:
done